static/scripts/calendar.js

```diff
@@ -304,4 +330,4 @@ $(document).ready(() => {
 			hideAreaByEvent(event);
 		}
 	});
-});
+});
```

Comment:
Please add one empty line at the end of the file to stop the Eslint complaining.

Reply:
Done